src/screens/transaction-details/transaction-details.tsx

```diff
@@ -54,9 +55,7 @@ export class TransactionDetailsComponent extends React.Component<
         const transaction = this.props.transaction;
         const account = this.props.account;
         const amount =
-            BLOCKCHAIN_INFO[account.blockchain].coin +
-            ' ' +
-            transaction.amount.toString().slice(0, 5);
+            BLOCKCHAIN_INFO[account.blockchain].coin + ' ' + formatAmountFromAccount(account);
```

Comment:
this doesn't seem to be right since amount is the amount of the transaction not account balance.